src/main/java/com/checkout/financial/Region.java

```diff
@@ -0,0 +1,12 @@
+package com.checkout.financial;
+
+import com.google.gson.annotations.SerializedName;
+
+public enum Region {
+    @SerializedName(value = "Domestic", alternate = {"DOMESTIC", "domestic"})
```

Comment:
Is the alternate something that happens during deserialization? Otherwise, we should keep only the value that appears on API reference.